src/components/Screens/HomeScreen.tsx

```diff
@@ -538,6 +539,11 @@ export default function HomeScreen() {
   const end = useSelector((state: RootState) => state.firefly.rangeDetails.end);
   const currentCode = useSelector((state: RootState) => state.currencies.currentCode);
   const dispatch = useDispatch<RootDispatch>();
+  const selectedTheme = useSelector((state: RootState) => state.configuration.selectedTheme);
```

Comment:
When user update the app via OTA new store key is `undefined` so we need a way to default to `gradientOrange` until the user select another theme:
 ```suggestion
  const selectedTheme = useSelector((state: RootState) => state.configuration.selectedTheme || 'gradientOrange');```

Reply:
True! Updated it. 55b5dc9f0af99d1f85951d3704744a830f4324ee a22bf175aec8ba26e175b450add53bc58bd3e3dc